source/scripting/Game.cs

```diff
@@ -156,17 +156,27 @@ public static float TimeScale
 			}
 		}
 		/// <summary>
-		/// returns the instantaneous Frames per Second
+		/// Gets the total number of frames that's been rendered in this session.
 		/// </summary>
-		public static float FPS
+		public static int FrameCount
 		{
 			get
 			{
-				return 1.0f / LastFrameTime;
+				return Function.Call<int>(Hash.GET_FRAME_COUNT);
 			}
 		}
 		/// <summary>
-		/// Gets how long(in seconds) the last frame was displayed before the next was rendered
+		/// Gets the current frame rate per second.
+		/// </summary>
+		public static float FrameRate
+		{
+			get
+			{
+				return 1.0f / DeltaTime;
```

Comment:
Why? The value of `FrameRate` is based on how the `Game` currently runs. Also, there are just like 2-3 props that would fit in a `Time`-class.